### PR TITLE
Chore/update snackbar

### PR DESCRIPTION
**Pull request by dawntancarrasco**
- udpate snackbar
- minor updates to datatable
- added inverse subtype for secondary and tertiary 

Screenshot 2023-11-16 at 3 24 13 PM
Screenshot 2023-11-16 at 3 24 50 PM

### CODE REVIEW

File: src/components/Button/Button.styles.ts [hunk 1]
```diff
@@ -301,6 +302,30 @@ const StyledButton = styled.button<{
...
+  ${({theme, subtype, variant, isLoading}) =>
+    variant === "secondary" &&
+    subtype === "inverse" &&
+    `
+    color: white;
+    background-color: transparent;
+    border: ${theme.borderWidth.md} solid white;
+
+    &:hover:not(:disabled) {
+      background-color: rgba(255, 255, 255, 0.05);
```
Comment (sbracegirdle) [inline src/components/Button/Button.styles.ts:314]: Question; Is there a theme prop we can use here?

File: src/components/DataTable/DataTable.styles.ts [hunk 2]
```diff
@@ -32,6 +32,11 @@ export const GcxDataTableTbody = styled.tbody`
 
 export const GcxDataTableTr = styled.tr`
   min-width: 0;
+  border-bottom: 1px solid #ebebeb;
```
Comment (sbracegirdle) [inline src/components/DataTable/DataTable.styles.ts:35]: Question; is there a theme prop we can use here?

**Review by sbracegirdle — APPROVED** [review]
LGTM, just a couple of questions

File: src/components/Button/Button.styles.ts (hunk 1 above)
Comment (dawntancarrasco) [reply src/components/Button/Button.styles.ts:314]: For this one, according to Lou there is no token we can use at the moment

File: src/components/DataTable/DataTable.styles.ts (hunk 2 above)
Comment (dawntancarrasco) [reply src/components/DataTable/DataTable.styles.ts:35]: Okay i will ask Lou if there is a token we can use then change it on my next pr :)

**Pull request merged**